### PR TITLE
edit community links section

### LINKED ISSUE #361: Follow up to #359
A few tweaks to the etcd.cn link on the community page:

- [ ] "version of this site" --> "translation of etcd.io"
- [ ] Put "(此...版)" inside / as part of the link text. Actually, use this text: "(本网站中文翻译)"
- [ ] ". Note that the Chinese language site is ..." --> "-- the website is ..."
- [ ] "by the etcd.io" --> "by etcd"

_Originally posted by @chalin in https://github.com/etcd-io/website/pull/359#discussion_r645120564_

### PR DESCRIPTION
**Pull request by kapunahelewong**
Edits copy in community links section

Fixes #361

### CODE REVIEW

File: content/en/community/_index.md
Comment (nate-double-u) [inline content/en/community/_index.md:93]: nit: remove .io when referring to the maintainers (like the copyright footer)

```suggestion
* [Chinese translation of etcd.io (本网站中文翻译)](https://etcd.cn). The website is an independent project and isn't officially endorsed by etcd maintainers.
```

**Review by chalin — APPROVED** [review]
LGTM (after Nate's nit is addressed), thanks!

**Review by nate-double-u — APPROVED** [review]
Thanks, looks good to me!

File: content/en/community/_index.md
Comment (wenjiaswe) [reply content/en/community/_index.md:93]: you are good! nits: "本网站的中文翻译" just to add one character in the middle.

**Pull request merged**
